package.json

```diff
@@ -34,16 +34,17 @@
   },
   "devDependencies": {
     "@catdad/to-bmp": "^1.1.0",
-    "chai": "^4.3.4",
+    "chai": "^4.4.1",
     "chalk": "^4.1.2",
     "css-font-local-cache": "^1.0.1",
     "electron": "^29.1.0",
     "electron-background": "^1.0.0",
     "electron-builder": "^24.13.3",
     "electronmon": "^2.0.2",
     "eslint": "^5.16.0",
+    "ffbinaries": "^1.1.6",
```

Comment:
This probably wasn't supposed to be listed here. It's already part of `video-tools`